pkg/storage/model_planStore.go

```diff
@@ -43,17 +43,20 @@ func (s *Store) ModelPlanCreate(logger *zap.Logger, plan *models.ModelPlan) (*mo
 	if err != nil {
 		logger.Error(
 			fmt.Sprintf("Failed to create model plan with error %s", err),
-			zap.String("user", models.ValueOrEmpty(plan.ModifiedBy)),
+			zap.String("user", models.ValueOrEmpty(plan.CreatedBy)),
 		)
 		return nil, err
 	}
 	retPlan := models.ModelPlan{}
 
+	plan.ModifiedBy = nil
+	plan.ModifiedDts = nil
```

Comment:
I wonder if it'd be more clear/reliable to just have the create SQL statement not even take `modifiedBy` and `modifiedDts` as params. That way, even if you somehow passed a `plan` object with those fields, they'd still get created `NULL`.

It's a tradeoff I think - makes it so that you're less flexible in creating objects exactly as you state them, but also means that you'll never accidentally write code that populates that field and sets it. 🤷  I could go either way - thoughts?

Reply:
I have been back and forth on this one SO MUCH, glad you brought it up. Truth is I'm just not sure, like you said either way is a pro/con. For now I think let's just go with this and have a call to change it if we want.